python/arepl_python_evaluator.py

```diff
@@ -65,35 +68,26 @@ def __init__(
         self.count = count
 
 
-if version_info[0] < 3 or (version_info[0] == 3 and version_info[1] < 5):
-    # need at least 3.5 for typing
-    exMsg = "Must be using python 3.5 or later. You are using " + str(version_info)
-    print(ReturnInfo("", "{}", None, None, exMsg))
-    raise Exception(exMsg)
-
-
 class ExecArgs(object):
 
     # HALT! do NOT change this without changing corresponding type in the frontend! <----
     # Also note that this uses camelCase because that is standard in JS frontend
-    def __init__(self, evalCode: str, savedCode="", filePath="", usePreviousVariables=False, *args, **kwargs):
+    def __init__(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing docstring in __init__ (D107)](https://app.codacy.com/manual/almenon214/AREPL-backend/pullRequest?prid=5418764)